### PR TITLE
Bugfix/subsidy bugfixes

### PR DESCRIPTION
**Pull request by benleolander**
Resolves IN-1170, 1171, 1172, 1173, 1174, 1175 and 1176

### CODE REVIEW

**Review by GlenHani — DISMISSED** [review]

File: INZFS.MVC/Controllers/FundApplicationController.cs [hunk 1]
```diff
@@ -341,6 +341,11 @@ public async Task<IActionResult> Save([Bind(Prefix = "submit.Publish")] string s
 
                 _session.Save(contentToSave);
 
+                if (currentPage.NextPageName != null)
```
Comment (sunilbeis) [inline INZFS.MVC/Controllers/FundApplicationController.cs:344]: This is   likely   to break some   scenarios.

Comment (sunilbeis) [reply INZFS.MVC/Controllers/FundApplicationController.cs:344]: The order of the next page check is important.

Comment (benleolander) [reply INZFS.MVC/Controllers/FundApplicationController.cs]: Done, moved it last, so it won't override anything that's already there

**Review by sunilbeis — APPROVED** [review]

**Review by lorenzoAtBEIS — APPROVED** [review]
Tested locally, all looking good. Approved

**Pull request merged**